metagraph/src/kmer/kmer_collector.cpp

```diff
@@ -127,22 +129,20 @@ void count_kmers(std::function<void(CallStringCount)> generate_reads,
 template <typename KMER, class KmerExtractor, class Container>
 KmerCollector<KMER, KmerExtractor, Container>
 ::KmerCollector(size_t k,
-                bool both_strands_mode,
+                Mode mode,
                 std::vector<typename Extractor::TAlphabet>&& filter_suffix_encoded,
                 size_t num_threads,
                 double memory_preallocated,
                 const std::filesystem::path &swap_dir,
-                size_t __attribute__((unused)) disk_cap_bytes,
-                bool canonical_only)
+                size_t __attribute__((unused)) disk_cap_bytes)
```

Comment:
```suggestion
                size_t /* disk_cap_bytes */)
```

Reply:
it's used for some cases (defined by `if constexpr(...)`)